typescript/src/offerings.ts

```diff
@@ -426,6 +466,10 @@ export interface PurchasesPromotionalOffer {
   readonly timestamp: number;
 }
 
+/**
+ * Enum with possible proration modes in a subscription upgrade or downgrade in the Play Store. Used only for Google.
+ * @public
```

Comment:
Well, these are actually warnings the library mentions. Basically this is only really useful if you use this tool to also generate docs (we don't generate docs for this library, and we use `tsdoc`/`@capacitor/docgen` in our other libraries). So we could just disable the warning... But I thought it would be good to follow the `TSDoc` standard, instead of just silencing the warning, but lmk if you feel strongly about it, and I would be ok to change it.